kuma/wiki/models.py

```diff
@@ -1327,15 +1325,11 @@ def populate_attachments(self, update_populated_field=False):
                 defaults={
                     'attached_by': revision.creator,
                     'name': revision.filename,
+                    'is_original': False,
```

Comment:
You're right, this shouldn't be here. I confused the semantics of `get_or_create` with that of `update_or_create`